app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -71,10 +77,11 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
             val bundle = Bundle()
             bundle.putString(USER_DATA, Json.encodeToString(user))
             MyFragment.beginTransaction<UserPlaylistsFragment>(supportFragmentManager, bundle)
-            if(fromGoogle){
+
+            /*if(fromGoogle){
```

Comment:
Linked to the previous comment, you can remove this aswell

Reply:
It was done between the time you read the code and wrote your comment.
If you look at the last commit, this line is no longer present.